src/index.ts

```diff
@@ -74,9 +74,6 @@ unhandled({
   showDialog: false,
 });
 
-// Disable Node options if the env var is set
-process.env.NODE_OPTIONS = '';
```

Comment:
Why was this removed?

Reply:
Are there any regressions caused by this change?

Reply:
@MulverineX Are there any regressions caused by this change?